lib/holdings/callnumber.rb

```diff
@@ -149,7 +149,12 @@ def stackmapable_library?
     end
 
     def stackmapable_location?
-      !Constants::HIDE_STACKMAP_LOCS.include?(home_location)
+      !Constants::STACKMAP_BLACKLIST[:global].include?(home_location)
```

Comment:
One suggestion here, which might simplify a bit.

What if we add a `blacklisted_stackmap_locations` method (or something like that) that did something along the lines of 

```ruby
Constants::STACKMAP_BLACKLIST[library] || Constants::STACKMAP_BLACKLIST[:global]
```

Then `stackmapable_location?` could basically be `blacklisted_stackmap_locations.include?(home_location)` and we could get rid of the `stackmapable_blacklisted?` method all-together (which also could simplify `stackmapable?` above).

Just a suggestion, if you still like this approach better, then I'm 👍